src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -339,18 +343,10 @@ public void initialize() {
 
   public void rezeroLift() {
     // Set arm encoder position from absolute
-    armEncoder.setPosition(
-        AngleUtil.wrapAngle(
-            armAbsoluteEncoder.getPosition() - Cal.Lift.ARM_ABSOLUTE_ENCODER_ZERO_POS_DEG));
+    armEncoder.setPosition(armAbsoluteEncoderChecker.getMedian());
 
     // Set elevator encoder position from absolute encoders
-    double elevatorDutyCycleEncodersDifferenceDegrees =
-        AngleUtil.wrapAngleAroundZero(
-            (elevatorLeftAbsEncoder.getPosition() - elevatorRightAbsEncoder.getPosition()));
-    elevatorLeftEncoder.setPosition(
-        (elevatorDutyCycleEncodersDifferenceDegrees
-                * Constants.Lift.ELEVATOR_MOTOR_ENCODER_DIFFERENCES_SCALAR_INCHES_PER_DEGREE)
-            - Cal.Lift.ELEVATOR_ABS_ENCODER_POS_AT_START_INCHES);
+    elevatorLeftEncoder.setPosition(elevatorLeftAbsEncoderChecker.getMedian());
```

Comment:
Same comment as for intake, we still need all the math, just replacing the `Encoder.getPosition()` bits with the filtered value